content/en/watchdog/insights.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Watchdog Insights
+kind: documentation
+description: View anomalies and outliers in context with Watchdog Insights
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-watchdog-insights-log-management/"
+  tag: "Blog"
+  text: "Augmented troubleshooting with Watchdog Insights"
+- link: "https://www.datadoghq.com/blog/watchdog-insights-apm/"
+  tag: "Blog"
+  text: "Automatically detect error and latency patterns with Watchdog Insights for APM"
+---
+
+## Overview
+
+Watchdog Insights inform you of anomalies and error outliers that relate to your current view, including any facets or search terms. During an incident, use Watchdog Insights to focus on the highest priority data and speed up investigation.
+
+Each Insight highlights one outlier or anomaly affecting a subset of users. Depending on the product area, Watchdog Insights can detect different types of anomalies.
+
+## Locations within Datadog
+
+Three areas within Datadog feature Watchdog Insights: Log Management, RUM, and APM.
+
+### Log Management
+
+In the Log Explorer, Watchdog Insights finds two types of insights:
+- [Log Anomaly Detection][1]
+- [Error Outliers][2]
+
+To locate Watchdog Insights in the Log Management UI, take the following steps:
+1. In the left navigation, hover over **Logs**
+2. Click **Search**
+
+The pink Watchdog Insights banner appears in the middle of your screen, above your logs. If Watchdog cannot find any issues, the banner is gray.
+
+For more information, see [Watchdog Insights for Logs][3].
+
+### RUM
+
+In the RUM explorer, Watchdog Insights finds two types of insights:
+- [Error Outliers][4]
+- [Latency Outliers][5]
+
+To locate Watchdog Insights in the RUM UI, take the following steps:
+1. In the left navigation, hover over **UX Monitoring**
+2. Click **Sessions & Replays**
+
+The pink Watchdog Insights banner appears in the middle of your screen, above your sessions. If Watchdog cannot find any issues, the banner is gray.
+
+For more information, see [Watchdog Insights for RUM][6].
+
+### APM
+
+Watchdog Insights appear on several pages within APM:
+- [Trace Explorer][7]
+- [Continuous Profiler][8]
+- Service Catalog (beta)
```

Comment:
@samirbrizini, when I go to the Service Catalog (https://app.datadoghq.com/services) in the demo org, I don't see a Watchdog Insights bar. Is it supposed to be there?